packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -724,9 +726,11 @@ export class BasicCrawler<
             await this.failedContextHandler(crawlingContext);
         } else {
             const { id, url, method, uniqueKey } = crawlingContext.request;
-            this.log.exception(
-                crawlingContext.error,
-                'Request failed and reached maximum retries',
+            this.log.error(
+                `Request failed and reached maximum retries. ${crawlingContext.error instanceof TimeoutError && this.log.getLevel() < 5
```

Comment:
Can we use a constant variable instead of `5`?

Reply:
I was pretty sure I fixed this one before, probably messed something up when I was making more changes. Fixed.